pkg/certificates/certificate_subject.go

```diff
@@ -58,6 +63,9 @@ func (c certificateSubject) generateCertificateTemplate(alternativeIP string) (x
 	if alternativeIP != "" {
 		ips = append(ips, net.ParseIP(alternativeIP))
 	}
+	for _, ip := range c.secondaryIPs {
```

Comment:
I guess that alternativeIP (service IP indeed) is pretty the same as this dataSubnetIP, so it the certification generation mechanism can be unified and this alternativeIP could be a part of secondaryIPs

Reply:
The difference between them is that serviceIP is common for all Pods of given type, and dataSubnetIP is specific for the Pod. I had a problem to maintain connection between pod and it's IP which prevented me from merging those two information in one. I also discussing about it with @kaweue here https://github.com/Juniper/contrail-operator/pull/398#discussion_r494900683